plugins/pulp_rpm/plugins/importers/yum/repomd/packages.py

```diff
@@ -37,7 +37,8 @@ def package_list_generator(xml_handle, package_tag, process_func=None):
     :rtype: generator
     """
     if process_func is None:
-        process_func = lambda x: x
+        def process_func(x):
+            return x
```

Comment:
Why this change? It's fine either way, but for some reason I feel compelled to say that this way it should have a docblock since it's now not anonymous. I think the lambda thing is simpler.